redbiom/commands/summarize.py

```diff
@@ -26,54 +24,43 @@ def summarize_caches():
 
 
 @summarize.command(name='metadata-category')
-@click.option('--category', required=True)
-@click.option('--counter', required=False, is_flag=True, default=False)
-@click.option('--descending', is_flag=True, required=False, default=False)
-@click.option('--dump', required=False, is_flag=True, default=False)
+@click.option('--category', required=True,
+              help="The metadata category (i.e., column) to summarize")
+@click.option('--counter', required=False, is_flag=True, default=False,
+              help="If true, obtain value counts")
+@click.option('--descending', is_flag=True, required=False, default=False,
+              help="If true, sort in descending order")
+@click.option('--dump', required=False, is_flag=True, default=False,
+              help="If true, print the sample information.")
 def summarize_metadata_category(category, counter, descending, dump):
     """Summarize the values within a metadata category"""
-    import redbiom
-    import redbiom.requests
-    import pandas as pd
-
-    get = redbiom.requests.make_get(redbiom.get_config())
+    if not counter and not dump:
+        click.echo("Please specify either --counter or --dump",
```

Comment:
That's a good suggestion but I'm not sure if it fits here well? This is a lack of a specified parameter, as opposed to incorrectly specifying a parameter. It might make sense for this to throw a `UsageError` though. Can you create an issue on resolving CLI exceptions?